specs/span-batches.md

```diff
@@ -247,14 +249,21 @@ Span-batches share the same queue with v0 batches: batches are processed in L1 i
 A set of modified validation rules apply to the span-batches.
 
 Rules are enforced with the [contextual definitions](./derivation.md#batch-queue) as v0-batch validation:
-`batch`, `epoch`, `inclusion_block_number`, `next_timestamp`, `next_epoch`, `batch_origin`
+`epoch`, `inclusion_block_number`, `next_timestamp`
+
+Definitions:
+
+- `batch` as defined in the [Span batch format section][span-batch-format].
+- `prev_l2_block` is the L2 block from the current safe chain,
+  whose timestamp is at `span_start.timestamp - l2_block_time`
 
 Span-batch rules, in validation order:
 
-- `batch.start_timestamp > next_timestamp` -> `future`: i.e. the batch must be ready to process.
-- `batch.start_timestamp < next_timestamp` -> `drop`: i.e. the batch must not be too old.
-- `batch.parent_check != safe_l2_head.hash[:20]` -> `drop`: i.e. the checked part of the parent hash must be equal
-  to the L2 safe head block hash.
+- `span_start.timestamp > next_timestamp` -> `future`: i.e. the batch must be ready to process.
+- `span_end.timestamp < next_timestamp` -> `drop`: i.e. the batch must have at least one new block.
+- If there's no `prev_l2_block` in the current safe chain -> `drop`: i.e. misaligned timestamp
```

Comment:
Why did we add this check?

Reply:
e.g. If block timestamps of the current safe chain are ...10, 12, 14, and the `span_start.timestamp` is 15, then we should find the `prev_l2_block` at timestamp 13 (15 - block time), but the block does not exist. so we have to drop this batch.

This new rule is to ensure that the timestamp of the span batch is well-aligned with the existing chain and block time.

Reply:
The second condition is comparing `span_end` instead of `span_start`. It might be a bit confusing, and I think you've made a mistake here.